### PR TITLE
New StatType condition

### PR DESCRIPTION
**Pull request by dm94**

### CODE REVIEW

File: src/main/java/com/github/manolo8/darkbot/config/actions/values/StatTypeValue.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            }
+        }
+
+        return null;
+    }
+
+    public enum StatData {
+        CURRENT,
+        EARNED,
+        SPENT;
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/config/actions/values/StatTypeValue.java:52]: Missing a "difference" or something like that, which is earned-spent, its what makes the most sense for graphs and overall "how much did the bot make"

File: src/main/java/com/github/manolo8/darkbot/config/actions/values/StatTypeValue.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        @Override
+        public String toString() {
+            return name().toLowerCase(Locale.ROOT).replace("_", "-");
+        }
+
+        public static StatData of(String sd) {
+            for (StatData sD : StatData.values()) {
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/config/actions/values/StatTypeValue.java:60]:
```suggestion
            for (StatData sd : StatData.values()) {
```

File: src/main/java/com/github/manolo8/darkbot/config/actions/values/StatTypeValue.java (hunk 1 above)
Comment (dm94) [reply src/main/java/com/github/manolo8/darkbot/config/actions/values/StatTypeValue.java:52]: Wouldn't that be EARNED?

Comment (Pablete1234) [reply src/main/java/com/github/manolo8/darkbot/config/actions/values/StatTypeValue.java:58]: Currently earned is only positive differences, so if you earn 100.000 uri, but spend 200.000 uri to gain it, stuff like uri/h will show positive, but the graph of overall uri should be going down (hence, difference, or earned - spent)

**Review by Pablete1234 — APPROVED** [review]

**Pull request merged**